core/src/services/rocksdb/backend.rs

```diff
@@ -118,31 +120,56 @@ impl kv::Adapter for Adapter {
     }
 
     async fn get(&self, path: &str) -> Result<Option<Vec<u8>>> {
-        self.blocking_get(path)
+        let cloned_self = self.clone();
+        let cloned_path = path.to_string();
+
+        task::spawn_blocking(move || cloned_self.blocking_get(cloned_path.as_str()))
+            .await
+            .map_err(new_task_join_error)
+            .and_then(|inner_result| inner_result)
```

Comment:
The same to https://github.com/apache/incubator-opendal/pull/3280#discussion_r1359134537

Reply:
Done!